### ISSUE TITLE
Warnings from mysqldump are being seen as an error

### ISSUE THREAD

**Issue by appliedzetetics** [julianpoemp/oc-backup#3]
``In this part of the code (the echo is something I inserted), it appears to be trying to exclude warning errors from mysqldump, but it's indicating that the dump experienced an error.

```
 if [ "${#stderr}" -gt 0 ]; then
    echo "mysqldump error: ${stderr}"
    if ! [[ "${stderr}" =~ ^mysqldump:[[:space:]]\[Warning\] ]]; then
      log "Error: SQL backup failed."
      log "_${stderr}_"
      errors_found=$((errors_found + 1))
    fi
  fi
```
Here is the logging output:

```
root@conf:/var/www/owncloud# ./oc_backup.sh
* 2023-02-15_14-52-45 | Start oc-backup v1.0.2...
* 2023-02-15_14-52-45 | -> Backup owncloud configuration file...
* 2023-02-15_14-52-45 | -> Read owncloud configuration file...
* 2023-02-15_14-52-45 | -> Enable maintenance mode...
* 2023-02-15_14-52-45 | -> Create backup folder
* 2023-02-15_14-52-45 | -> Create database backup...
* 2023-02-15_14-52-46 | mysqldump: [Warning] Using a password on the command line interface can be insecure.
mysqldump error: mysqldump: [Warning] Using a password on the command line interface can be insecure.
* 2023-02-15_14-52-46 | Error: SQL backup failed.
* 2023-02-15_14-52-46 | _mysqldump: [Warning] Using a password on the command line interface can be insecure._
* 2023-02-15_14-52-46 | -> Create zip archive of owncloud files...

```
Am I missing something, here?

**Comment by julianpoemp** [comment]
It seems that the regular expression isn't recognized correctly. Which type of Linux do you use? It could be similar to the issue before about enabling/disabling mentainance mode.

**Comment by appliedzetetics** [comment]
It's running Ubuntu 22.04...

**Comment by julianpoemp** [comment]
> It's running Ubuntu 22.04...

I'll have a detailed look at it tomorrow or saturday. I think I have to use another regular expression matching method.

Thank you for your feedback!

**Comment by appliedzetetics** [comment]
Thank you for your responsiveness, and what promises to be an excellent tool!

**Comment by julianpoemp** [comment]
Hi @appliedzetetics , I just released version 1.0.3 that fixes this issue. Thanks again! :)

**Comment by appliedzetetics** [comment]
Many thanks! I shall test it later today.
